percy/providers/generic_provider.py

```diff
@@ -42,6 +46,15 @@ def _get_tag(self, **kwargs):
         }
 
     def _get_tiles(self, **kwargs):
+        fullpage_ss = kwargs.get('fullpage_screenshot', False)
+        tile_count = kwargs.get('num_of_tiles', 5)
+        if fullpage_ss:
+            if self.supports(self.metadata.remote_url):
+                data = self.execute_percy_screenshot(self.metadata.device_height, tile_count, self.metadata.scale_factor)
```

Comment:
This function should not be called in base class as its defined in sub class. Rather override get_tiles in subclass